buildpipeline/pipelines.json

```diff
@@ -132,7 +132,8 @@
         "TreatWarningsAsErrors": "false"
       },
       "BuildParameters": {
-        "PB_BuildType": "Release"
+        "PB_BuildType": "Release",
+        "EnforcePGO": "enforcepgo"
```

Comment:
You should use a `PB_` prefix (matching in the definition jsons). It isn't always used in CoreCLR, but it helps distinguish values set by the PipeBuild infrastructure from environment variables.